src/index.js

```diff
@@ -3,14 +3,18 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import styles from '../lib/styles.css';
 
-const Icon = ({icon, flex}) =>
-  <span className={
-    classnames(styles.icon, styles[icon], {[`${styles.flex}`]: flex})
-  } />;
-
+const Icon = ({icon, ariaHidden, title, flex}) =>
+  <span
+    className={
+      classnames(styles.icon, styles[icon], {[`${styles.flex}`]: flex})
+    }
+    aria-hidden
```

Comment:
props in react are camelCased

Reply:
not aria: https://reactjs.org/docs/dom-elements.html